package.json

```diff
@@ -32,15 +32,14 @@
   "license": "MIT",
   "devDependencies": {
     "@biomejs/biome": "2.2.4",
-    "@tailwindcss/cli": "^4.1.13",
     "@types/express": "^5.0.3",
     "@types/node": "^20.11.17",
     "@types/nunjucks": "^3.2.6",
     "@vitest/coverage-v8": "^3.2.4",
     "@vitest/ui": "^3.2.4",
     "chokidar": "^3.6.0",
     "daisyui": "^5.1.26",
-    "tailwindcss": "^4.1.13",
+    "tailwindcss": "^3.4.17",
```

Comment:
How come we are downgrading tailwind? 

Wouldn't we have more access to nice new features in 4.1.13?